backend/fms_core/viewsets/sample.py

```diff
@@ -90,68 +75,13 @@ def get_serializer_class(self):
     def get_renderer_context(self):
         context = super().get_renderer_context()
         if self.action == 'list_export':
-            fields = SampleExportSerializer.Meta.fields
+            fields = SampleSerializer.Meta.fields
```

Comment:
This is something that is directly tied to the Export Serializer it should not be moved to a normal serializer.